.github/workflows/test.yml

```diff
@@ -0,0 +1,133 @@
+name: Github Test CI
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@master
+        with:
+          submodules: 'recursive'
+
+      - name: Read .nvmrc
+        run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
+        id: nvm
+
+      - name: Use Node.js (.nvmrc)
+        uses: actions/setup-node@master
+        with:
+          node-version: "${{ steps.nvm.outputs.NVMRC }}"
+
+      - name: Cache node modules
```

Comment:
Ako funguje ta cache? Ja som sa docital ze musis manualne skipnut step:
```
if: steps.restore-cache.outputs.cache-hit != 'true'
```
[Vid moj PR](https://github.com/priznanie-digital/priznanie-digital/pull/166/files#diff-fe8421955fd596131bb6f1b78984b2fbR47).

Lebo teraz ked sa pozeram na ten job, tak `install` aj `build` step prebehnu ako keby cache nebola. A inicializacia cache prida 3 + 3 sekundy - nie je to vela, ale vyzera ze je to zbytocne.

Reply:
Ja myslim ze by to malo fungovat aj bez toho, ale nie som si ulne isty.
Ta cache len cachuje .npm priecinok, cize `npm install` moze pouzit packages z disku namiesto stahovania.

Tu to vidis ze cache bola restornuta https://github.com/priznanie-digital/priznanie-digital/pull/169/checks?check_run_id=826968457#step:5:15

Aspon v robote to mamte takto vsade :)

Reply:
Skus dat prec tu cache a instalacia bude trvat rovnako dlho ako s nou 🤷‍♂️ Neviem ci je instalacia packagov z repozitara z netu rovnako rychla ako z lokalneho zdroja. Vidim ze to pise o restornuti cache, ale nevidim ziadne zrychlenie 😢

Reply:
tak potom neviem 🤷‍♂️
mozme skusit yarn, ten je podla mna rychlejsi, aj ked asi to je jedno, tych par sekund co sa usetri podla mna zato nestoji